docs/release-notes.md

```diff
@@ -10,6 +10,90 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and selected important fixes. Release notes published since April 2022 include Semgrep CLI, CI, and Semgrep App updates. Release notes published since April 2022 also include updates from more versions collected together.
 
+## July 2022
+
+### Semgrep App
+
+#### Additions
+- Semgrep App now integrates with Slack through a Slack app. To create a new integration, go to **Settings > Integrations > Add Integration > Slack**. Previously, Semgrep App used Slack webhooks.
+- Enable autofix for all of your Projects (repositories connected to Semgrep App) by clicking on **Settings > Deployment > Autofix**.
+
+#### Changes
+- Clicking on the Project Name in the Projects page now takes you to that project's Findings page. Click the gear icon at the end of the Project's row to go to the project's Settings page.
+- Semgrep App detects additional environment variables depending on your provider. This simplifies the creation and committing of the configuration file when adding a new Project (repository) in Semgrep App.
+- UI and UX improvements to **Scan new project** workflow.
+
+### Semgrep CLI
+
+These release notes include upgrades for all versions ranging between 0.102.0 and 0.107.0.
+
+#### Additions
+
+- Semgrep in CI:
+  - Fail-open support: Added `--suppress-errors` and `--no-suppress-errors` (the default is `--no-suppress-errors`).
+  - Support for podman environments!
+  - Semgrep in CI does not block builds on triage ignored issues.
+  - The timeout for Git commands Semgrep runs is now configurable. To configure the timeout, set the `SEMGREP_GIT_COMMAND_TIMEOUT` environment variable. The time unit used as value for this key is in seconds. The default value is 300.
+  - The `SEMGREP_GHA_MIN_FETCH_DEPTH` environment variable lets you set how many commits `semgrep ci` fetches from the remote at the minimum when calculating the merge-base in GitHub Actions. Having more commits available helps Semgrep determine what changes came from the current pull request, fixing issues where Semgrep would otherwise report findings that were not touched in a given pull request. This value is set to 0 by default. (Issue [#5664](https://github.com/returntocorp/semgrep/pull/5664))
+
+- Extract mode:
+  - New Semgrep CLI experimental extract mode. This mode runs a Semgrep rule on a codebase and extracts code from matches, treating it as a different language. This allows you to supplement an existing set of rules, for example, for JavaScript, by writing additional rules to find JavaScript in files of a different language than JavaScript. For example, JavaScript code in HTML or template files. While this is somewhat possible with `metavariable-pattern`, this reduces the work from an M \* N problem to an M \+ N. To know more about extract mode, see [Extract mode](experiments/extract-mode.md) documentation.
```

Comment:
Too many occurrences of JavaScript in a single sentence?
What about removing `for example, for JavaScript` ?

Reply:
```suggestion
  - New Semgrep CLI experimental extract mode. This mode runs a Semgrep rule on a codebase and extracts code from matches, treating it as a different language. This allows you to supplement an existing set of rules, for example, by writing additional rules to find JavaScript in files of a different language than JavaScript. Among many possible use cases, this enables you to write rules for HTML code in JavaScript code or in template files. While this is somewhat possible with `metavariable-pattern`, this reduces the work from an M \* N problem to an M \+ N. To know more about extract mode, see [Extract mode](experiments/extract-mode.md) documentation.
```